src/crimson/net/ProtocolV1.cc

```diff
@@ -699,6 +699,8 @@ ceph::bufferlist ProtocolV1::do_sweep_messages(
     if (session_security) {
       session_security->sign_message(msg.get());
     }
+    logger().debug("{} --> #{} === {} ({})",
```

Comment:
could you attach a sample of the logging messages before and after this change?

Reply:
Before:
```
[osd.0(server1) v2:127.0.0.1:9010/1] <== [osd.3 127.0.0.1:0/0@1234] #487 === ping magic: 0 v1 (2) // [osd.0(server1) v2:127.0.0.1:9010/1 >> osd.3 127.0.0.1:0/0@1234]
[osd.0(server1) v2:127.0.0.1:9010/1] --> [osd.3 127.0.0.1:0/0@1234] #487 === ping magic: 0 v1 (2) // [osd.0(server1) v2:127.0.0.1:9010/1 >> osd.3 127.0.0.1:0/0@1234]
```
After:
```
[osd.0(server1) v2:127.0.0.1:9010/1 >> osd.3 127.0.0.1:0/0@1234] --> #488 === ping magic: 0 v1 (2)
[osd.0(server1) v2:127.0.0.1:9010/1 >> osd.3 127.0.0.1:0/0@1234] <== #489 === ping magic: 0 v1 (2)
```

Reply:
the logging message from async messenger looks like

```
-- [v2:192.168.2.8:40116/0,v1:192.168.2.8:40117/0] --> [v2:192.168.2.8:40116/0,v1:192.168.2.8:40117/0] -- log(1 entries from seq 1 at 2019-07-17T11:05:17.828311+0800) v1 -- 0x5595f16e8900 con 0x5595f07b6000

-- [v2:192.168.2.8:40116/0,v1:192.168.2.8:40117/0] <== mgr.4104 192.168.2.8:0/48923 12050 ==== mgrbeacon mgr.x(be0de6ce-83
dd-44b0-b21b-1d01b502f9e9,4104, [v2:192.168.2.8:6800/48923,v1:192.168.2.8:6801/48923], 1) v8 ==== 10759+0+0 (crc 0 0 0) 0x5595f151db80 con 0x5595f1b9d180
```

@cyx1231st i am fine with change. but it would be great if we could have a more intuitive way to represent the sender / recipient like what we have without this change and in async messenger.